modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/display/context/JournalDisplayContext.java

```diff
@@ -544,6 +546,20 @@ public String getKeywords() {
 		return _keywords;
 	}
 
+	public JournalArticle getLatestArticle(JournalArticle journalArticle) {
+		try {
+			return JournalArticleLocalServiceUtil.getLatestArticle(
```

Comment:
@zoltancsaszi this is not needed. the local service has a fetch counterpart, why not use that?